ansible/roles/empire/templates/etc/empire/env/empire.env

```diff
@@ -1,27 +0,0 @@
-EMPIRE_PORT=8080
```

Comment:
I think we also need to update the [readme](https://github.com/remind101/empire_ami/blob/master/README.md)